public/app/features/dashboard-scene/serialization/transformToV2TypesUtils.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  DashboardLink as OldDashboardLink,
+  VariableHide as OldVariableHide,
+  VariableRefresh as OldVariableRefresh,
+  VariableSort as OldVariableSort,
+} from '@grafana/schema';
+import {
+  DashboardCursorSync,
+  defaultDashboardSpec,
+  DashboardLinkType,
+  DashboardLink,
+  defaultVariableHide,
+  defaultVariableRefresh,
+  defaultVariableSort,
+  VariableHide,
+  VariableRefresh,
+  VariableSort,
+} from '@grafana/schema/dist/esm/schema/dashboard/v2alpha0/dashboard.gen';
+
+export function transformCursorSynctoEnum(cursorSync?: number): DashboardCursorSync {
```

Comment:
the argument should be not number I think but rather the v1 DashboardCursorSync type?